docs/api/system-preferences.md

```diff
@@ -380,6 +380,20 @@ You can use the `setAppLevelAppearance` API to set this value.
 Sets the appearance setting for your application, this should override the
 system default and override the value of `getEffectiveAppearance`.
 
+### `systemPreferences.promptTouchID(reason)` _macOS_
```

Comment:
My concern here is that this method **by itself** doesn't actually provide any true security for user-data / app-data.  Apps would have to implement some kind of locking mechanism based on a `true` / `false` value here through something like `keytar`.  (Store an encryption key in `keytar` and only fetch it if `promptTouchID` resolves with `true`).

Not exactly sure how I would phrase it, but I feel as though we need to do due diligence here and warn people people they have to implement real security / encryption based around this method.  E.g. The way native apps would do this is by setting [`kSecAccessControlUserPresence `](https://developer.apple.com/documentation/security/secaccesscontrolcreateflags/ksecaccesscontroluserpresence?language=objc) on their keychain entry so that reading it would auto-prompt for biometrics.